extensions/resteasy-classic/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/AuthenticationCompletionExceptionMapper.java

```diff
@@ -4,13 +4,18 @@
 import javax.ws.rs.ext.ExceptionMapper;
 import javax.ws.rs.ext.Provider;
 
+import org.jboss.logging.Logger;
+
 import io.quarkus.security.AuthenticationCompletionException;
 
 @Provider
 public class AuthenticationCompletionExceptionMapper implements ExceptionMapper<AuthenticationCompletionException> {
 
+    private static final Logger log = Logger.getLogger(AuthenticationCompletionExceptionMapper.class.getName());
```

Comment:
Did you enable debug for this category?

Reply:
Yeah, I realized I missed it so converted to Draft :-), checking now with one of the tests expecting 401